### PR TITLE
Add io.github.weclaw1.ScoreTracker

### PR DESCRIPTION
**Pull request by weclaw1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.github.weclaw1.ScoreTracker.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+app-id: com.github.weclaw1.ScoreTracker
```
Comment (hfiguiere) [inline com.github.weclaw1.ScoreTracker.yaml:1]: as per the documentation, theis app-id should start with `io.github.`

File: com.github.weclaw1.ScoreTracker.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - install -Dm755 ./target/release/score-tracker -t /app/bin/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.svg -t /app/share/icons/hicolor/scalable/apps/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker-symbolic.svg -t /app/share/icons/hicolor/symbolic/apps/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.desktop -t /app/share/applications/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.metainfo.xml -t /app/share/metainfo/
+  sources:
+  - cargo-sources.json
+  - type: git
+    url: https://github.com/weclaw1/score-tracker.git
+    tag: 1.0
```
Comment (bbhtt) [inline com.github.weclaw1.ScoreTracker.yaml:32]:
```suggestion
    tag: '1.0'
```
Fixes a small warning

File: com.github.weclaw1.ScoreTracker.yaml (hunk 1 above)
Comment (weclaw1) [reply com.github.weclaw1.ScoreTracker.yaml]: Fixed, I also released a new version with updated app id

File: com.github.weclaw1.ScoreTracker.yaml (hunk 2 above)
Comment (weclaw1) [reply com.github.weclaw1.ScoreTracker.yaml]: Fixed

**Comment by bbhtt** [comment]
bot, build io.github.weclaw1.ScoreTracker

**Pull request closed without merging**